File: app/src/main/java/com/lkms/data/model/java/PeerReview.java
package com.lkms.data.model.java;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Đại diện cho một hàng trong bảng "PeerReview".
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class PeerReview {

    @SerializedName("reviewId")
    private Integer reviewId;

    // Khóa ngoại đến Project
    @SerializedName("projectId")
    private Integer projectId;

    @SerializedName("startTime")
    private String startTime;

    @SerializedName("endTime")
    private String endTime;

    @SerializedName("detail")
    private String detail;
}

File: app/src/main/java/com/lkms/data/model/kotlin/PeerReview.kt
package com.lkms.data.model.kotlin

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class PeerReview(
    @SerialName("reviewId")
    val reviewId: Int? = null,

    // Khóa ngoại đến Project
    @SerialName("projectId")
    val projectId: Int? = null,

    @SerialName("startTime")
    val startTime: String? = null,

    @SerialName("endTime")
    val endTime: String? = null,

    @SerialName("detail")
    val detail: String? = null
)